atom/browser/api/atom_api_app.cc

```diff
@@ -183,7 +187,7 @@ void App::OnFinishLaunching() {
   auto browser_context = static_cast<AtomBrowserContext*>(
       AtomBrowserMainParts::Get()->browser_context());
   auto handle = Session::CreateFrom(isolate(), browser_context);
-  default_session_.Reset(isolate(), handle.ToV8());
+  g_wrap_session.Run(handle.ToV8());
```

Comment:
if `_setWrapSession` is implemented in `atom_api_session.cc` then have to expose `session` as a builtin module to use on the js side, is that correct ? and yeah that would help us calling it in one place.